### PR TITLE
feat: use Paginator to split rep history

### PR DESCRIPTION
**Pull request by Josh-Cena**
Let's split the message so Discord doesn't choke on long histories :D

### CODE REVIEW

**Comment by tjjfvi** [comment]
Is this tested? I'm not sure that option will work on embeds

**Comment by futurGH** [comment]
Yeah, it doesn't. The Discord.js splitMessage method only accepts strings.

**Comment by Josh-Cena** [comment]
Okay, going back to this–does anyone have ideas about:

1. How am I going to test this?
2. What fix should I apply? Maybe let the bot send a plain text string, or use reactions to tell the bot to switch pages? (I've observed this behavior with the TeXit bot)

Test2

**Comment by robertt** [comment]
I made [this](https://github.com/typescript-community/community-bot/blob/old/src/utils/Paginator.ts) for the old version of the bot - might be worth porting it over and using it in this case?

**Comment by Josh-Cena** [comment]
Oh that looks cool, will look into this this weekend 👍

**Comment by Josh-Cena** [comment]
That looks better...

**Comment by robertt** [comment]
This looks great! Has it been tested?

**Comment by Josh-Cena** [comment]
How do I test it? I'm really curious. Do I have to self-host it?

**Comment by robertt** [comment]
I can test it if you want! Otherwise, you'll have to setup a local discord bot and run it using the docker-compose

**Comment by Josh-Cena** [comment]
Yeah, I probably can't spare doing that right now. Would appreciate it if you can test it 👍

File: src/util/sendPaginatedMessage.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+	await message.react(emojis.first);
+	await message.react(emojis.back);
+	await message.react(emojis.stop);
+	await message.react(emojis.next);
+	await message.react(emojis.last);
+
+	const collector = message.createReactionCollector(
+		(reaction, user) =>
+			reaction.me &&
```
Comment (robertt) [inline src/util/sendPaginatedMessage.ts:40]: Can you remove this like here? Checking for reaction.me is deprecated and is no longer needed in the version of discordjs that we use.

**Review by robertt — APPROVED** [review]
Works great! One minor thing:

File: src/util/sendPaginatedMessage.ts (hunk 1 above)
Comment (Josh-Cena) [reply src/util/sendPaginatedMessage.ts:40]: Oh, sure

**Comment by robertt** [comment]
Thanks!

**Pull request merged**